dataland-keycloak-adapter/src/main/kotlin/org/dataland/keycloakAdapter/auth/DatalandJwtAuthentication.kt

```diff
@@ -19,7 +19,7 @@ class DatalandJwtAuthentication(private val jwt: Jwt) : DatalandAuthentication()
         return realmRoles?.map { SimpleGrantedAuthority(it as String) } ?: listOf()
     }
 
-    override fun getCredentials(): Jwt {
-        return jwt
```

Comment:
Why this change?

Reply:
Because the community manager needs to execute "getCredentials" to get the token of the requesting user.
Reason: The community manager re-uses that token to ask the backend if it knows a company with the provided LEI, PermID or ISIN

Reply:
Was just wondering because now the function returns a subset of the information carried by the jwt token. Was this function also just at other places and if so did we never needed other information stored in the jwt token?

Reply:
Ok, closed